### PR TITLE
Add libx265/3.4

### PR DESCRIPTION
**Pull request by theirix**
Specify library name and version:  **libx265/3.4**

- Version bump (no upstream build changes)
- Fix homepage
- Changed cmake name to x265

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libx265/all/conanfile.py [hunk 1]
```diff
@@ -113,6 +113,8 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
 
     def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "x265"
```
Comment (madebr) [inline recipes/libx265/all/conanfile.py:116]: The problem here is that libx265's cmake script only creates and installs a pc files.
The policy of cci is to only set the names of the *official* installed cmake config scripts.

https://github.com/videolan/x265/blob/master/source/CMakeLists.txt

Comment (theirix) [reply recipes/libx265/all/conanfile.py:116]: I supposed that a cmake project name and pc file was enough to deduce cmake name, find scripts for libx265 or x265 are not widespread but it seems that x265 is a little more popular. If such a policy exists, no problem, I will drop it.

Comment (SpaceIm) [reply recipes/libx265/all/conanfile.py]: I agree with @madebr, CCI recipes shouldn't suggest non official CMake names.

Comment (theirix) [reply recipes/libx265/all/conanfile.py]: Sounds reasonable. Dropped this chunk.

**Review by madebr — APPROVED** [review]

**Comment by SpaceIm** [comment]
LGTM. Could update PR title?

**Review by SpaceIm — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Pull request merged**